### ISSUE TITLE
Manually loading checkpoints

### ISSUE THREAD

**Issue by r-khanna** [rosinality/style-based-gan-pytorch#115]
I want to manually load the pre-trained weights to the generator and use it. However, the images generated after loading the checkpoints are not right. The samples generated using generate.py(same checkpoints) are correct. 

Please let me know what am I doing wrong. Below is my code - 

```
import torch
import torch.nn as nn
import torchvision
import torch.optim as optim
import model
import student_model
import matplotlib.pyplot as plt
import math


n_batch = 1
dim_inp = 512

gen_teacher = model.StyledGenerator(512).cuda()

path_checkpoint = './checkpoint/stylegan-256px-new.model'
checkpoint = torch.load(path_checkpoint)
gen_dict = checkpoint['generator']
gen_teacher.load_state_dict(gen_dict)

noise = torch.randn(n_batch,dim_inp).cuda()

out_teacher = gen_teacher(input=noise,noise=None,step=6)
plt.imshow(torchvision.transforms.ToPILImage()(out_teacher[0].cpu()))
```

Here is the output - 
![image](https://user-images.githubusercontent.com/52320802/109847852-a8cda780-7c75-11eb-8218-383397443f95.png)

Thanks in advance :)

**Comment by rosinality** [comment]
The value range of output images is (-1, 1). I think ToPILImage() expects it to be (0, 1). So you need to normalize image values (-1, 1) to (0, 1).

**Comment by r-khanna** [comment]
> 
> 
> The value range of output images is (-1, 1). I think ToPILImage() expects it to be (0, 1). So you need to normalize image values (-1, 1) to (0, 1).

That works. Thanks :)
